spec/fixture/.atom-build.error-match-errorFile.json

```diff
@@ -0,0 +1,5 @@
+{
+	"cmd": "echo 'line:3,column:8' && return 1",
+	"errorMatch": "line:(?<line>\\d+),column:(?<col>\\d+)",
+  "errorFile": ".atom-build.json"
+}
```

Comment:
Indentation is weird in this file. Also, you could consider if you want to define this in the test instead of having a file:

```javascript
fs.writeFileSync(directory + '.atom-build.json', JSON.stringify({
  cmd: "echo 'line:3,column:8' && return 1",
  errorMatch: "line:(?<line>\\d+),column:(?<col>\\d+)",
  errorFile: ".atom-build.json"
}).;
```

I've been leaning more towards that lately as the whole test is encapsulated in that one `it`.